### PR TITLE
feat: add apisix dashboard deb support

### PR DESCRIPTION
**Pull request by EnableAsync**
issue: #62

### CODE REVIEW

**Comment by nic-chen** [comment]
@EnableAsync please resolve conflicts

**Comment by EnableAsync** [comment]
@nic-chen Updated, thanks for the reminder. 😆

**Comment by imjoey** [comment]
@EnableAsync Hi, how are you doing? 😄 Sorry for being late. As PR #86 has been merged, could you please continue to reduce the CI files newly added within this PR? Your patience and understanding are much appreciated.

**Comment by EnableAsync** [comment]
@imjoey Understood, I will reduce the CI files in this PR.

**Comment by EnableAsync** [comment]
ping @spacewander @nic-chen @imjoey for review at your free time. Thanks.

**Review by imjoey — APPROVED** [review]
/lgtm. @EnableAsync  Nice PR, thank you very much.

File: .github/workflows/package-apisix-dashboard-deb-ubuntu20.04.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          ETCD_ADVERTISE_CLIENT_URLS: http://0.0.0.0:2379
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: install dependencies
+        run: |
+          sudo apt-get install -y make build-essential
+
+      - name: run apisix-dashboard packaging test
```
Comment (nic-chen) [inline .github/workflows/package-apisix-dashboard-deb-ubuntu20.04.yml:35]:
```suggestion
      - name: package apisix-dashboard
```

File: .github/workflows/package-apisix-dashboard-deb-ubuntu20.04.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        run: |
+          docker exec ubuntu20.04Instance bash -c "dpkg -i /apisix-dashboard/apisix-dashboard_${DASHBOARD_VERSION}-0~ubuntu20.04_amd64.deb"
+          docker logs ubuntu20.04Instance
+          docker exec ubuntu20.04Instance bash -c "cd /usr/local/apisix/dashboard/ && nohup ./manager-api &"
+
+      - name: run test cases
+        run: |
+          code=$(curl -k -i -m 20 -o /dev/null -s -w %{http_code} http://127.0.0.1:9000)
+          if [ ! $code -eq 200 ]; then
+              echo "failed: failed to install Apache APISIX Dashboard by rpm"
```
Comment (nic-chen) [inline .github/workflows/package-apisix-dashboard-deb-ubuntu20.04.yml:53]:
```suggestion
              echo "failed: failed to install Apache APISIX Dashboard by deb"
```

File: utils/install-common.sh [hunk 3]
```diff
@@ -114,6 +114,51 @@ install_apisix() {
     rm -rf /tmp/build/output/apisix/usr/local/apisix/deps/lib/luarocks/rocks-5.1/apisix/master-${iteration}/doc
 }
 
+install_golang() {
+    wget https://dl.google.com/go/go1.15.2.linux-amd64.tar.gz
```
Comment (nic-chen) [inline utils/install-common.sh:118]: I think we could use 1.16

**Review by nic-chen — APPROVED** [review]

File: .github/workflows/package-apisix-dashboard-deb-ubuntu20.04.yml (hunk 1 above)
Comment (EnableAsync) [reply .github/workflows/package-apisix-dashboard-deb-ubuntu20.04.yml]: Updated.

File: .github/workflows/package-apisix-dashboard-deb-ubuntu20.04.yml (hunk 2 above)
Comment (EnableAsync) [reply .github/workflows/package-apisix-dashboard-deb-ubuntu20.04.yml]: Updated.

File: utils/install-common.sh (hunk 3 above)
Comment (EnableAsync) [reply utils/install-common.sh]: Updated, thanks for your suggestion.

**Comment by imjoey** [comment]
@EnableAsync Great PR, much appreciated. @nic-chen Thank you very much for the review. 😄

**Pull request merged**